docs/extended_api/thread_scopes.md

```diff
@@ -1,56 +1,61 @@
 ---
-grand_parent: API
-parent: Synchronization Library
+parent: Extended API
 nav_order: 0
 ---
 
 # Thread Scopes
 
-Standard C++ presents a view that the cost to synchronize threads is uniform
-  and low.
-CUDA C++ is different: the overhead is low among threads within a block, and
-  high across arbitrary threads in the system.
-
-To bridge these two realities, libcu++ introduces the concept of **thread scopes**,
-  to the Standard's concurrency facilities in the `cuda::` namespace, while
-  retaining the syntax and semantics of Standard C++ by default.
-A thread scope specifies the group of threads that can synchronize with each
-  other using a concurrency primitive, such as an `atomic` or a `barrier`.
-
-## Synopsis
-
-```c++
+```cuda
 namespace cuda {
 
+// Header `<cuda/atomic>`.
```

Comment:
Please make the synopsis consistent with other pages (see pipelines.md for example).

Reply:
This one os the one exception I'd make to the other ones: here we are spanning a lot of headers and just providing an overview of the scope-extended APIs across all of them. I expect it to look very messy if we split this snippet.